inst/tests/other.Rraw

```diff
@@ -155,15 +155,6 @@ if (loaded[["knitr"]]) {
   test(11, kable(DT), output="x.*y.*1.*2")
 }
 
-# for plm package
```

Comment:
perhaps we could offer to add this test to plm suite instead?

Reply:
palm does not depends on DT so I don't think it is worth it